content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -208,70 +237,81 @@ my-nginx-o0ef1                 1/1       Running   0          29m       nginx
 ```shell
 kubectl get pods -lapp=guestbook,role=slave
 ```
-```shell
+
+```none
 NAME                          READY     STATUS    RESTARTS   AGE
 guestbook-redis-slave-2q2yf   1/1       Running   0          3m
 guestbook-redis-slave-qgazl   1/1       Running   0          3m
 ```
 
 ## Canary deployments
 
-Another scenario where multiple labels are needed is to distinguish deployments of different releases or configurations of the same component. It is common practice to deploy a *canary* of a new application release (specified via image tag in the pod template) side by side with the previous release so that the new release can receive live production traffic before fully rolling it out.
+Another scenario where multiple labels are needed is to distinguish deployments of different
+releases or configurations of the same component. It is common practice to deploy a *canary* of a
+new application release (specified via image tag in the pod template) side by side with the
+previous release so that the new release can receive live production traffic before fully rolling
+it out.
 
 For instance, you can use a `track` label to differentiate different releases.
 
 The primary, stable release would have a `track` label with value as `stable`:
 
-```yaml
-     name: frontend
-     replicas: 3
-     ...
-     labels:
-        app: guestbook
-        tier: frontend
-        track: stable
-     ...
-     image: gb-frontend:v3
+```none
```

Comment:
Is there an issue displaying the code as YAML?

Reply:
The three dots make the snippet an invalid YAML.